modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,512 @@
+		_actionDropdownItems = actionDropdownItems;
+	}
+
+	public void setDisabled(Boolean disabled) {
+		_disabled = disabled;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setGroupName(String groupName) {
+		_groupName = groupName;
+	}
+
+	public void setHref(String href) {
+		_href = href;
+	}
+
+	public void setIcon(String icon) {
+		_icon = icon;
+	}
+
+	public void setImageAlt(String imageAlt) {
+		_imageAlt = imageAlt;
+	}
+
+	public void setImageSrc(String imageSrc) {
+		_imageSrc = imageSrc;
+	}
+
+	public void setInputName(String inputName) {
+		_inputName = inputName;
+	}
+
+	public void setInputValue(String inputValue) {
+		_inputValue = inputValue;
+	}
+
+	public void setLabels(List<LabelItem> labels) {
+		_labels = labels;
+	}
+
+	public void setName(String name) {
+		_name = name;
+	}
+
+	public void setSelectable(Boolean selectable) {
+		_selectable = selectable;
+	}
+
+	public void setSelected(Boolean selected) {
+		_selected = selected;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setSpritemap(String spritemap) {
+		_spritemap = spritemap;
+	}
+
+	public void setSubtitle(String subtitle) {
+		_subtitle = subtitle;
+	}
+
+	public void setUserCard(UserCard userCard) {
+		_userCard = userCard;
+	}
+
+	public void setUserColorClass(String userColorClass) {
+		_userColorClass = userColorClass;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_disabled = false;
+		_groupName = null;
+		_href = null;
+		_icon = null;
+		_imageAlt = null;
+		_imageSrc = null;
+		_inputName = null;
+		_inputValue = null;
+		_labels = null;
+		_name = null;
+		_selectable = false;
+		_selected = false;
+		_spritemap = null;
+		_subtitle = null;
+		_userCard = null;
+		_userColorClass = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/UserCard";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("actions", _actionDropdownItems);
+		props.put("description", _subtitle);
+		props.put("disabled", _disabled);
+		props.put("href", _href);
+		props.put("userImageAlt", _imageAlt);
+		props.put("userImageSrc", _imageSrc);
+		//props.put("checkboxProps", {name: inputName, value: inputValue});
+		props.put("labels", _labels);
+		props.put("name", _name);
+		props.put("selectable", _selectable);
+		props.put("selected", _selected);
+		props.put("spritemap", _spritemap);
+		props.put("userDisplayType", _userColorClass);
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		cssClasses.add("card-type-asset");
+		cssClasses.add("user-card");
+		cssClasses.add("form-check");
+		cssClasses.add("form-check-card");
+		cssClasses.add("form-check-top-left");
+
+		if (!_selectable) {
+			cssClasses.add("card");
```

Comment:
While I'm looking at it, sort classes, `card` and `user-card` seem out of order :p